packages/blocks/src/root-block/widgets/element-toolbar/more-menu/button.ts

```diff
@@ -29,9 +32,7 @@ const BUILT_IN_GROUPS = [
 export class EdgelessMoreButton extends WithDisposable(LitElement) {
   override render() {
     const context = new ElementToolbarMoreMenuContext(this.edgeless);
-    const groups = context.config.configure(
-      BUILT_IN_GROUPS.map(group => ({ ...group, items: [...group.items] }))
-    );
+    const groups = context.config.configure(cloneGroups(BUILT_IN_GROUPS));
```

Comment:
This will reconfigure the group every time the button rendered, can we cache the behavior?

Reply:
Good catch.
I will fix it in next PR.

Reply:
See #8089